lib/index.js

```diff
@@ -71,7 +71,7 @@ internals.implementation = (server, options) => {
                 }
 
                 if (!isValid) {
-                    return reply(Boom.unauthorized('Bad token', options.tokenType), { credentials, artifacts });
+                    return reply(Boom.unauthorized(null, options.tokenType), { credentials, artifacts });
```

Comment:
I think this would be a breaking change to suddenly allow chaining by default. Also, I prefer everything to be locked down by default, and optional where possible so I think the best solution here is to make this optional.

Here a new option `allowChaining` which it defaults to `false`. If `false` add the `Bad Token` string here. If `allowChaining` is true, pass `null` here.